java/connector-node/risingwave-connector-service/src/main/java/com/risingwave/connector/source/common/PostgresValidator.java

```diff
@@ -180,7 +180,8 @@ private void validateTableSchema() throws SQLException {
         // check primary key
         // reference: https://wiki.postgresql.org/wiki/Retrieve_primary_key_columns
         try (var stmt = jdbcConnection.prepareStatement(ValidatorUtils.getSql("postgres.pk"))) {
-            stmt.setString(1, this.schemaName + "." + this.tableName);
```

Comment:
Why not just quote the object name? 
`String.format("\"%s\".\"%s\"", this.schemaName, this.tableName)`

Reply:
Can be another choice. I noticed that other sql statements use schemaName and tableName, so I think we should keep them consistent. Do you prefer `"\"%s\".\"%s\""`?

Reply:
The change to the validate sql needs to join two more system tables, I think it can be avoided.

Reply:
updated.

Reply:
Slightly prefer to keep consistent with other places, unless the cost of join is high.

Strictly speaking the manual quoting above is wrong, but I do believe no one would challenge this in practice:
```
test=# create table "my_test""table" ("Foo" int); -- two quotes required
CREATE TABLE
test=# select relname from pg_class where relname ^@ 'my_te';
    relname    
---------------
 my_test"table
(1 row)
```
([pgdoc](https://www.postgresql.org/docs/16/functions-string.html#:~:text=Embedded%20quotes%20are%20properly%20doubled))
> Embedded quotes are properly doubled